lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```diff
@@ -0,0 +1,189 @@
+
+    describe('EditTeam', function () {
+        var teamEditView,
+            teamsUrl = '/api/team/v0/teams/',
+            teamsData = {
+                id: null,
+                name: "TeamName",
+                is_active: null,
+                course_id: "a/b/c",
+                topic_id: "awesomeness",
+                date_created: "",
+                description: "TeamDescription",
+                country: "c",
+                language: "a",
+                membership: []
+            },
+            verifyValidation = function (requests, fieldsData) {
+                _.each(fieldsData, function (fieldData) {
+                    teamEditView.$(fieldData[0]).val(fieldData[1]);
+                });
+
+                teamEditView.$('.create-team.form-actions .action-primary').click();
+
+                var message = teamEditView.$('.wrapper-msg');
+                expect(message.is(':visible')).toBeTruthy();
+                expect(message.find('.title').text().trim()).toBe("Your team could not created!");
+                expect(message.find('.copy').text().trim()).toBe(
+                    "Check the highlighted fields below and try again."
+                );
+
+                _.each(fieldsData, function (fieldData) {
+                    if(fieldData[2] === 'error') {
+                        expect(teamEditView.$(fieldData[0].split(" ")[0] + '.error').length).toBe(1);
+                    } else if(fieldData[2] === 'success') {
+                        expect(teamEditView.$(fieldData[0].split(" ")[0] + '.error').length).toBe(0);
+                    }
+                });
+
+                expect(requests.length).toBe(0);
+            },
+            expectContent = function (selector, text) {
+                expect(teamEditView.$(selector).text().trim()).toBe(text);
+            },
+            verifyDropdownData = function (selector, expectedItems) {
+                var options = teamEditView.$(selector)[0].options;
+                var renderedItems = $.map(options, function( elem ) {
+                    return [[elem.value, elem.text]];
+                });
+                for (var i = 0; i < expectedItems.length; i++) {
+                    expect(renderedItems).toContain(expectedItems[i]);
+                }
+            };
+
+        beforeEach(function () {
+            setFixtures('<div class="teams-content"></div>');
+            spyOn(Backbone.history, 'navigate');
+            teamEditView = new TeamEditView({
+                el: $('.teams-content'),
+                teamParams: {
+                    teamsUrl: teamsUrl,
+                    courseId: "a/b/c",
+                    topicId: 'awesomeness',
+                    topicName: 'Awesomeness',
+                    languages: [['a', 'aaa'], ['b', 'bbb']],
+                    countries: [['c', 'ccc'], ['d', 'ddd']]
+                }
+            }).render();
+        });
+
+        it('can render itself correctly', function () {
```

Comment:
Thanks for clarifying the refactor, which I hadn't noticed. I still think that you are testing way to much here. Since this view renders an new model, there's no model data to be rendered. You're basically testing the template, but this should be a test of the backbone code. Therefore you should only check that the edit view's `$el` is present and the `$el` of each child view (such as the field views) are present.